packages/discovery-styles/scss/global-carbon-styles.scss

```diff
@@ -0,0 +1,14 @@
+// You only need to import this file if your app is not already directly importing these carbon files
```

Comment:
If we like this approach, I'd write up a better statement here that links to docs including this: https://github.com/carbon-design-system/carbon/blob/master/docs/guides/sass.md#optimizing-your-sass-builds